jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/dtls/DtlsConfig.kt

```diff
@@ -23,4 +24,16 @@ class DtlsConfig {
     val handshakeTimeout: Duration by config {
         "jmt.dtls.handshake-timeout".from(JitsiConfig.newConfig)
     }
+
+    val ciphersSuites: List<Int> by config {
+        "jmt.dtls.cipher-suites".from(JitsiConfig.newConfig).convertFrom<List<String>> {list ->
+            list.map { it.toBcCipherSuite() }
+        }
+    }
+
+    companion object {
+        val config = DtlsConfig()
+    }
 }
+
+private fun String.toBcCipherSuite(): Int = CipherSuite::class.java.getDeclaredField(this).getInt(null)
```

Comment:
This will throw `NoSuchFieldException` if the field name is spelled wrong - is that good enough, or do we want a cleaner error message?